src/components/Header.js

```diff
@@ -26,6 +26,7 @@ function networkCustomGroupData(networkMap) {
       notification: `Switched to <b>${name}</b>.`,
       url: networkMap[name].url,
       networkId: networkMap[name]?.networkId || name,
+      chainId: networkMap[name].chainId,
```

Comment:
maybe can modify, because if there is no value may lead to a black screen:
chainId: networkMap[name]?.chainId